project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -677,15 +682,139 @@ function c4m_user_access_denied() {
 /**
  * Implements hook_menu_alter().
  *
- * Make the user/%user/contact page just a callback.
+ * Make the user/%user/contact page.
  */
 function c4m_user_menu_alter(&$items) {
   $items['user/login'] = array(
     'title' => 'Please log in to continue',
   );
   $items['user/%user/contact'] = array(
     'type' => MENU_CALLBACK,
+    'title' => 'User contact form',
+    'description' => 'User contact form.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('c4m_user_contact_form', 1),
+    'access arguments' => array('administer users'),
+  );
+}
+
+/**
+ * Create 'Contact User' form.
+ */
+function c4m_user_contact_form($form, &$form_state, $account) {
+  global $user;
+  $form_state['user'] = $account;
+
+  $form['c4m_user_from'] = array(
+    '#type' => 'textfield',
+    '#title' => t('From'),
+    '#description' => t("From address"),
+    '#size' => 30,
+    '#default_value' => $user->mail,
+    '#disabled' => TRUE,
+  );
+
+  $form['c4m_user_subject'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Subject'),
+    '#description' => t("Enter mail subject."),
+    '#size' => 10,
   );
+
+  $form['c4m_user_body'] = array(
+    '#title' => t('Body'),
+    '#description' => t('Enter mail body'),
+    '#type' => 'text_format',
+    '#format' => 'full_html',
+  );
+
+  $form['c4m_user_checkbox'] = array(
```

Comment:
Change the id to `c4m_contact_user_self_copy`.

also change the prefix of all fields from `c4m_user` to `c4m_contact_user`

thanks.